source/agora/consensus/data/Block.d

```diff
@@ -56,6 +57,8 @@ public struct BlockHeader
     /// Schnorr multisig of all validators which signed this block
     public Signature signature;
 
+    /// Enrolled validators
+    public Enrollment[] enrollments;
```

Comment:
Shouldn't this be part of the hash?

Reply:
I thought of a better way, but now I'd better put it in and have a hashpart.